scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -1,9 +1,11 @@
+# distutils: language = c++
```

Comment:
Do we really need this now that we use `meson`?

Reply:
Removed.